src/components/GlobalPreferences/Notifications/UnlockNotifications.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import { Button, GU, IconConnect, textStyle, useTheme } from '@aragon/ui'
+import emailNotifcationIllustration from '../../../../src/assets/emailNotifications.svg'
+
+const UnlockNotifications = React.memo(function UnlockNotifications({
+  compactMode,
+  needsUnlockSettings,
+  onUnlock,
+}) {
+  const theme = useTheme()
+
+  return (
+    <div
+      css={`
+        padding: ${3 * GU}px;
+      `}
+    >
+      <div
+        css={`
+          display: flex;
+          flex-direction: column;
+          text-align: center;
+        `}
+      >
+        <div>
+          <img
+            src={emailNotifcationIllustration}
+            width={141}
+            height={141}
+            alt=""
+          />
+        </div>
+        <h3
+          css={`
+            ${textStyle('title2')};
+            margin-top: ${3 * GU}px;
+          `}
+        >
+          Unlock notifications settings
+        </h3>
+        <div
+          css={`
+            display: flex;
+            flex-direction: column;
+            background: ${needsUnlockSettings ? '' : theme.background};
+            padding: ${3 * GU}px ${compactMode ? 3 : 13 * GU}px;
```

Comment:
```suggestion
            padding: ${3 * GU}px ${(compactMode ? 3 : 13) * GU}px;
```

The previous version was not working as you might have expected in `compactMode` :).